src/components/PeopleTable.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import PeopleRow from './PeopleRow';
+
+const createTableHeaders = (people) => {
+  if (people.length === 0) {
+    return [{ name: 'There are no people' }];
+  }
+
+  return (
+    Object.keys(people[0]).map(
+      key => ({
+        code: key, name: key[0].toUpperCase() + key.slice(1,),
+      })
+    )
+  );
+};
+
+class PeopleTable extends Component {
+  state = {
+    selectedId: 0,
+  }
+
+  setSelectedId = (id) => {
+    this.setState({
+      selectedId: id,
+    });
+  };
+
+  render = () => {
+    const { people, sortTable } = this.props;
+    const { selectedId } = this.state;
+    const TABLE_HEADERS = createTableHeaders(people);
```

Comment:
```suggestion
    const tableHeaders = createTableHeaders(people);
```
З великих буков пишеться коли ця константа по за функцією і може імпортуватися в інші файли, тут же це константа створена функцією для внутрішнього користування

Reply:
Теперь понимаю. Спасибо)